tests/select_for_update/tests.py

```diff
@@ -182,7 +255,7 @@ def test_for_update_requires_transaction_only_in_execution(self):
 
     @skipUnlessDBFeature('supports_select_for_update_with_limit')
     def test_select_for_update_with_limit(self):
-        other = Person.objects.create(name='Grappeli')
+        other = Person.objects.create(name='Grappeli', born=self.city1, died=self.city2)
```

Comment:
I've left it for now, because making the fields nullable runs into errors about the interaction of SELECT FOR UPDATE and outer joins.

I can try harder if necessary :)